shell/package.json

```diff
@@ -41,6 +41,7 @@
     "@rancher/icons": "2.0.29",
     "@types/is-url": "1.2.30",
     "@types/node": "16.4.3",
+    "@types/semver": "^7.5.8",
```

Comment:
I don't think we need this, or?

Reply:
I think we do @cnotv , as Stackstate extension is not using the `semver` library explicitly but it throws error on build related to that

Reply:
Let's keep it then. Thanks for the update!